.readthedocs.yaml

```diff
@@ -13,7 +13,6 @@ build:
     python: "3.11"
 
 python:
-  system_packages: False
```

Comment:
this was removed from RTD 2023-08-29 

https://blog.readthedocs.com/drop-support-system-packages/

We need  this change to build the docs

Reply:
thanks - I'll add a new commit